README.md

```
@@ -1,7 +1,26 @@
+```
+
+Alternative method
+
 ```
-pip install opencv-python
+pip install opencv-python requirements
+```
+
+If that doesnt work use `pip3`
+
+```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)